### PR TITLE
Added destructive_lightning.sc

### PR DESCRIPTION
**Pull request by Madff386**
Once you start it any lightning that strikes, either natural, summoned or a channeling trident, a TNT and fireball will be summoned to create a small crater where the lightning strikes.

### CODE REVIEW

**Comment by Ghoulboy78** [comment]
use `__on_tick()` event, not the loop.

**Comment by Madff386** [comment]
Ok I updated it to use the `__on_tick()` event instead of the loop, and also added functionality to stop it without reloading.

File: programs/survival/destructive_lightning.sc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+__on_tick() -> (
+    if(global_started == 1,
+    e = entity_list('lightning_bolt'); //finds all lightning bolts
+        for(e,
+            look = l(0.0, -1.0, 0.0);
+            fireball_pos = (_ ~ 'pos')+look;  //finds lightning bolt position
+            if (global_last != query(_, 'uuid'),  //only summons one fireball per bolt
+                spawn('tnt', (_ ~ 'pos'));
+                game_tick(50);
```
Comment (gnembon) [inline programs/survival/destructive_lightning.sc:25]: why do you need to run a game tick here?

Comment (Madff386) [reply programs/survival/destructive_lightning.sc:25]: Forgot to remove it when I copied the code out of the loop and into the __on_tick event

**Pull request merged**